src/Interface/BusinessProfileInterface/BusinessProfileInterfaceUtils.res

```diff
@@ -102,3 +128,309 @@ let mapJsonToBusinessProfileV2 = (values): profileEntity => {
+    always_collect_billing_details_from_wallet_connector: profileEntity.always_collect_billing_details_from_wallet_connector,
+    is_connector_agnostic_mit_enabled: profileEntity.is_connector_agnostic_mit_enabled,
+    is_click_to_pay_enabled: profileEntity.is_click_to_pay_enabled,
+    authentication_product_ids: profileEntity.authentication_product_ids,
+    outgoing_webhook_custom_http_headers: profileEntity.outgoing_webhook_custom_http_headers,
+    is_auto_retries_enabled: profileEntity.is_auto_retries_enabled,
+    max_auto_retries_enabled: profileEntity.max_auto_retries_enabled,
+    metadata: profileEntity.metadata,
+    force_3ds_challenge: profileEntity.force_3ds_challenge,
+    is_debit_routing_enabled: profileEntity.is_debit_routing_enabled,
+    acquirer_configs: profileEntity.acquirer_configs,
+    merchant_category_code: profileEntity.merchant_category_code,
+    is_network_tokenization_enabled: profileEntity.is_network_tokenization_enabled,
+    always_request_extended_authorization: profileEntity.always_request_extended_authorization,
+    always_enable_overcapture: profileEntity.always_enable_overcapture,
+    is_manual_retry_enabled: None,
+    collect_shipping_details_from_wallet_connector_if_required: None,
+    collect_billing_details_from_wallet_connector_if_required: None,
+  }
+}
+let mapV2toCommonType: HSwitchSettingTypes.profileEntity => HSwitchSettingTypes.commonProfileEntity = profileEntity => {
+  {
+    profile_id: profileEntity.profile_id,
+    merchant_id: profileEntity.merchant_id,
+    profile_name: profileEntity.profile_name,
+    return_url: None,
+    payment_response_hash_key: None,
+    webhook_details: {
+      webhook_version: None,
+      webhook_username: None,
+      webhook_password: None,
+      webhook_url: None,
+      payment_created_enabled: None,
+      payment_succeeded_enabled: None,
+      payment_failed_enabled: None,
+    },
+    authentication_connector_details: None,
+    collect_shipping_details_from_wallet_connector: None,
+    always_collect_shipping_details_from_wallet_connector: None,
+    collect_billing_details_from_wallet_connector: None,
+    always_collect_billing_details_from_wallet_connector: None,
+    is_connector_agnostic_mit_enabled: None,
+    is_click_to_pay_enabled: None,
+    authentication_product_ids: None,
+    outgoing_webhook_custom_http_headers: None,
+    is_auto_retries_enabled: None,
+    max_auto_retries_enabled: None,
+    metadata: None,
+    force_3ds_challenge: None,
+    is_debit_routing_enabled: None,
+    acquirer_configs: None,
+    merchant_category_code: None,
+    is_network_tokenization_enabled: None,
+    always_request_extended_authorization: None,
+    always_enable_overcapture: None,
+    is_manual_retry_enabled: None,
+    collect_shipping_details_from_wallet_connector_if_required: None,
+    collect_billing_details_from_wallet_connector_if_required: None,
```

Comment:
why are all these mapped to `None`, shouldn't these be taken from `profileEntity` ?